### ISSUE TITLE
[BUG] System Error - Critical error - League of Legends

### ISSUE THREAD

**Issue by ssjp4** [RiotGames/developer-relations#453]
**Bug Description**  
Entering the loading screen of a game it gives a "Critical error"

**Problem Description**  
After choosing an ARURF match, selecting champ runes etc, match going to start the loading and it gives an error.

**Expected Result**  
Entering the match.
**Actual Result**  
Crtical error with the request to create a .tmp file to help dev team.

**Player Impact**  
Can't play.

** Severity**
Very high

** Region** 
EUW.

[PKMDD7F.zip](https://github.com/RiotGames/developer-relations/files/6004972/PKMDD7F.zip)

**Comment by geozukunft** [comment]
This GitHub is related to Riot Games API issues. Please refer to Riot Games support https://support.riotgames.com/ for ingame issues.

**Comment by BlueDab** [comment]
Yeah same here just started happening

**Comment by Hi-Ray** [comment]
Hey there!

This isn't the page to ask for that for help with things like that
please contact support through support@riotgames.com or go to

https://support.riotgames.com/hc/en-us


We ask that you close this issue as this is where issues regarding
the Riot Games API (A tool which provides data to sites like op.gg)
are posted.

Posting with issues regarding support or gameplay issues will simply
be ignored and most likely be closed anyway, It is advised you contact
support with the details provided above.

Kind Regards

TLDR: No help/ support at all will be given and this issue will be closed
